ajax/libs/keras-js/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "keras-js",
+  "version": "0.3.0",
+  "filename": "keras.js",
+  "description": "Run Keras models (tensorflow backend) in the browser, with GPU support.",
+  "homepage": "https://transcranial.github.io/keras-js/",
+  "keywords": [
+    "keras",
+    "deep",
+    "learning",
+    "machine",
+    "neural",
+    "networks",
+    "javascript",
+    "webgl",
+    "gpu"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/transcranial/keras-js"
```

Comment:
As told by @PeterDaveHello in another PR, the repository url being valid is enough. It doesn't need to match the upstream in the format it's written. :)